### PR TITLE
Add callerid support for txlogz and querylogz

### PR DESCRIPTION
**Pull request by guokeno0**

### CODE REVIEW

**Comment by guokeno0** [comment]
@sougou 

This is the back-port from internal commit

File: go/vt/tabletserver/querylogz_test.go [hunk 1]
```diff
@@ -44,11 +45,18 @@ func TestQuerylogzHandler(t *testing.T) {
 	logStats.CacheMisses = 2
 	logStats.CacheInvalidations = 3
 	logStats.TransactionID = 131
+	logStats.ctx = callerid.NewContext(
+		context.TODO(),
```
Comment (sougou) [inline go/vt/tabletserver/querylogz_test.go]: This should be context.Background(). Apparently TODO is to be used only for non-text code.

**Comment by sougou** [comment]
lgtm. one comment.

File: go/vt/tabletserver/querylogz_test.go (hunk 1 above)
Comment (guokeno0) [inline go/vt/tabletserver/querylogz_test.go]: Done

**Pull request merged**